File: app/src/main/java/com/frankly/randomdog/ui/recentlygenerateddog/RecentlyGeneratedDogViewModel.java
package com.frankly.randomdog.ui.recentlygenerateddog;

import androidx.lifecycle.ViewModel;

import com.frankly.randomdog.model.DogModel;

import java.util.ArrayList;

/**
 * @author: ArshVardhanAtreya <arshvardhan@yahoo.co.in>
 */

public class RecentlyGeneratedDogViewModel extends ViewModel {

    private ArrayList<DogModel> dogsList;

    public ArrayList<DogModel> getDogsList() {
        return dogsList;
    }

    public void setDogsList(ArrayList<DogModel> dogsList) {
        this.dogsList = dogsList;
    }
}

File: app/src/main/java/com/randomdog/ui/recentlygenerateddog/viewmodel/RecentlyGeneratedDogViewModel.kt
package com.randomdog.ui.recentlygenerateddog.viewmodel

import androidx.lifecycle.ViewModel
import com.randomdog.model.DogModel
import java.util.*

/**
 * @author: ArshVardhanAtreya <arshvardhan></arshvardhan>@yahoo.co.in>
 */
class RecentlyGeneratedDogViewModel : ViewModel() {
    private var dogsList: ArrayList<DogModel?>? = null
    fun getDogsList(): ArrayList<DogModel?>? {
        return dogsList
    }

    fun setDogsList(dogsList: ArrayList<DogModel?>?) {
        this.dogsList = dogsList
    }
}